javascript/javascript1/week2/week2-guzide/flight-booking/FligthBooking.js

```diff
@@ -0,0 +1,30 @@
+function getFullname(
+  firstname,
+  surname,
+  useFormalName = false,
+  gender = "female"
+) {
+  firstname = firstname || "";
+  surname = surname || "";
+
+  if (!firstname && !surname) {
+    return "Please provide your FIRST NAME and SURNAME!";
+  }
+
+  let nameTitle = "";
+  if (useFormalName) {
+    if (gender === "female") {
+      nameTitle = "Lady ";
+    } else if (gender === "male") {
+      nameTitle = "Lord ";
+    } else {
+      nameTitle = "";
+    }
+  }
+
+  return console.log(nameTitle + firstname + " " + surname);
```

Comment:
```suggestion
const fullName = nameTitle + firstname + " " + surname
  return fullName;
```

Here also you can return the compujted value from the funcion and then in the script you can write `console.log(getFullname("Benjamin", "Hughes", true, ""))` to see the value that funcion returns

Reply:
It helped me a lot to solve the previous issue as well. Thank you for the feedback.